ui/apos/components/AposImportModal.vue

```diff
@@ -245,7 +245,6 @@ export default {
     },
     async runImport() {
       if (!this.universalModuleAction) {
-        console.error('AposImportModal: No module action found');
```

Comment:
this developer-oriented ("technical") log could be useful?
if so, i'd suggest keeping it and disabling the rule for this line

Reply:
IDK, I feel like it's globally a bad practice to have developer-oriented logs in frontend since final users can see it..
Unless it gives configuration instructions?